backend/agentd/session.go

```diff
@@ -206,6 +194,12 @@ func (s *Session) Start() error {
 	org, env := s.cfg.Organization, s.cfg.Environment
 	agentID := s.cfg.AgentID
 
+	defer func() {
+		if err != nil {
```

Comment:
I'm probably missing it but where do we assign something to `err`?

Reply:
@palourde it's the named return variable on line 187

Reply:
Oh my bad, I got confused by how the PR presented the changes!